CHANGELOG.md

```diff
@@ -21,6 +21,8 @@ and this project adheres to
   - [#2686](https://github.com/iovisor/bpftrace/pull/2686)
 - Support targeting all running processes for USDTs
   - [#2734](https://github.com/iovisor/bpftrace/pull/2734)
+- Add a `jiffies` builtin for advanced usages
+  - [#2769](https://github.com/iovisor/bpftrace/pull/2769)
```

Comment:
I just did a new release of bpftrace, please move this to the "Unreleased" section of the changelog.